src/components/layout/MobileNavigation.vue

```diff
@@ -0,0 +1,175 @@
+<template>
+  <div
+    class="mobile-navigation clearfix"
+    :class="[
+      (open ? 'open' : ''),
+      ($route.name !== 'home' ? 'sticky' : ''),
+    ]"
+    v-show="$route.name !== 'home' || !isMobileNavigationHidden"
+  >
+    <div
+      class="close-navigation"
+      v-if="open"
+      @click="openNavigation(false)"
+    >
+      &#x2715;
+    </div>
+    <navigation/>
+    <div class="mobile-actions">
+      <img
+        class="tip"
+        :src="emptyHeart"
+        @click="toggleTipModal(true)"
+        v-if="!open && $route.name === 'home'"
+        v-show="Object.keys(account).length && balance"
+      >
+      <img
+        class="trigger-search"
+        :src="iconSearch"
+        @click="toggleMobileNavigation(true)"
+        v-if="!open && $route.name === 'home'"
+      >
+      <img
+        class="trigger-navigation"
+        :src="iconMobileMenu"
+        @click="openNavigation(true)"
+        v-if="!open"
+      >
+    </div>
+    <div class="footer text-center" v-if="open">
+      SuperHero is Open Source
+      <router-link class="ae" to="/terms">
+        Terms
+      </router-link>
+      <router-link class="ae" to="/privacy">
+        Privacy
+      </router-link>
+    </div>
+    <mobile-send-tip-modal></mobile-send-tip-modal>
+  </div>
+</template>
+
+<script>
+import { mapGetters, mapActions } from 'vuex';
+import Navigation from './Navigation.vue';
+import iconMobileMenu from '../../assets/iconMobileMenu.svg';
```

Comment:
i think this import is useful only for dynamic assets. You can simply put that in src="" (without `:`).